<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>686</width>
    <height>487</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CorrelationAnalysis</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>:/res/icon.png</normaloff>:/res/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QDockWidget" name="filesData">
   <property name="windowIcon">
    <iconset>
     <normaloff>:/res/icon.png</normaloff>:/res/icon.png</iconset>
   </property>
   <property name="allowedAreas">
    <set>Qt::LeftDockWidgetArea|Qt::RightDockWidgetArea</set>
   </property>
   <property name="windowTitle">
    <string>File's data</string>
   </property>
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionFilesDataVisible"/>
   <addaction name="actionStart"/>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>686</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionShowToolBar"/>
    <addaction name="actionFilesDataVisible"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <action name="actionFilesDataVisible">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/options.png</normaloff>:/res/options.png</iconset>
   </property>
   <property name="text">
    <string>Show file's data window</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/load.png</normaloff>:/res/load.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionStart">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/go.png</normaloff>:/res/go.png</iconset>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </action>
  <action name="actionShowToolBar">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show tool bar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>filesData</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionFilesDataVisible</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>39</x>
     <y>189</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>toolBar</sender>
   <signal>visibilityChanged(bool)</signal>
   <receiver>actionShowToolBar</receiver>
   <slot>setChecked(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>202</x>
     <y>36</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionFilesDataVisible</sender>
   <signal>triggered(bool)</signal>
   <receiver>filesData</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>39</x>
     <y>189</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionShowToolBar</sender>
   <signal>triggered(bool)</signal>
   <receiver>toolBar</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>36</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
